share/decorators.py

```diff
@@ -0,0 +1,37 @@
+import json
+import jwt
+​
+from django.http import JsonResponse
+​
+from user.models import (
+                        User,
+                    )
+from my_settings import (
+                        SECRET,
+                        ALGORITHM,
+                    )
+​
```

Comment:
```suggestion
from user.models import (
    User,
)
from my_settings import (
    SECRET,
    ALGORITHM,
)​
```
간격 조정해주시면 더 좋을 것 같아요~

Reply:
승진님이 수정할 예정입니다!